contracts/src/v0.8/l2ep/arbitrum/ArbitrumValidator.sol

```diff
@@ -79,6 +79,15 @@ contract ArbitrumValidator is ITypeAndVersion, AggregatorValidatorInterface, Sim
    */
   event L2WithdrawalRequested(uint256 indexed id, uint256 amount, address indexed refundAddr);
 
+  /**
+   * @notice emitted when a validation is executed successfully
+   * @param previousRoundId previous aggregator OCR round id
+   * @param previousAnswer previous aggregator answer
+   * @param currentRoundId current aggregator OCR round id
+   * @param currentAnswer new aggregator answer - value of 1 considers the service offline.
+   */
+  event ValidatedStatus(uint256 previousRoundId, int256 previousAnswer, uint256 currentRoundId, int256 currentAnswer);
```

Comment:
Please use BaseValidator.ValidatedStatus instead of redefining

Reply:
Fixed, thanks!